src/index.js

```diff
@@ -1,8 +1,11 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import App from './components/App';
+// import App from './components/App';
```

Comment:
It’s better to delete commented code. If you need to review a previous version of your code, use git history.